pandas/io/excel/_base.py

```diff
@@ -852,6 +853,14 @@ def __init__(self, path_or_buffer, engine=None):
                 ext = os.path.splitext(str(path_or_buffer))[-1]
                 if ext == ".ods":
                     engine = "odf"
+
```

Comment:
```suggestion
if ext == ".xls":
    engine = "xlrd"
```
Given that openpyxl doesn't support old `.xls` files.

Reply:
Added